README.md

```diff
@@ -212,6 +212,11 @@ metadata][service-configuration-metadata].
 * **log** (optional) - 
 see [logger configuration](./logger/README.md)
 
+* **max_message_size_in_mb** (optional; default: `4MB`) - 
```

Comment:
default should be just `4` - otherwise people will think they need to specify the `MB` suffix in the config value.